### PR TITLE
[13.0][IMP] stock_cubiscan: Handle unit without packaging

### PR DESCRIPTION
**Pull request by grindtildeath**
Depends on :
 - [x] https://github.com/OCA/product-attribute/pull/597

### CODE REVIEW

File: stock_cubiscan/wizard/cubiscan_wizard.py [hunk 1]
```diff
@@ -23,46 +23,90 @@ class CubiscanWizard(models.TransientModel):
...
+                    ),
+                }
+            )
+        return [vals]
+
+    def _prepare_packaging_lines(self):
+        vals_list = []
+        packaging_types = self.env["product.packaging.type"].search([])
+        for seq, pack_type in enumerate(packaging_types):
+            pack = self.env["product.packaging"].search(
```
Comment (simahawk) [inline stock_cubiscan/wizard/cubiscan_wizard.py:75]: if this wizard is used very often, I'd drop the search and add a read_group by product filtering on all pkg types before the loop.
Then in the loop you can lookup by product id.

File: stock_cubiscan/wizard/cubiscan_wizard.py [hunk 2]
```diff
@@ -23,46 +23,90 @@ class CubiscanWizard(models.TransientModel):
...
+                        self.product_id.product_width, cubiscan_mm_uom
+                    ),
+                    "height": product_dimension_uom._compute_quantity(
+                        self.product_id.product_height, cubiscan_mm_uom
+                    ),
+                }
+            )
+        return [vals]
+
+    def _prepare_packaging_lines(self):
```
Comment (simahawk) [inline stock_cubiscan/wizard/cubiscan_wizard.py:71]: docstring?

File: stock_cubiscan/wizard/cubiscan_wizard.py (hunk 1 above)
Comment (grindtildeath) [reply stock_cubiscan/wizard/cubiscan_wizard.py:75]: This part wasn't changed in this PR but only moved to a dedicated function. Sorry but I don't want to mess with its behaviour now...

**Comment by guewen** [comment]
Rebased and applied pre-commit

**Review by guewen — APPROVED** [review]

**Review by simahawk — APPROVED** [review]

**Pull request merged**